src/pcapi/routes/serialization/bookings_serialize.py

```diff
@@ -13,10 +13,15 @@
 
 def serialize_booking(booking: Booking) -> dict:
     booking_id = humanize(booking.id)
-    user_email = booking.user.email
+    if booking.educationalBookingId is not None:
```

Comment:
```suggestion
    is_educational_booking = booking.educationalBookingId is None
    if is_educational_booking:
```

Et inverser le if pour éviter d'avoir un premier if qui est en `not` et un deuxième (l23) qui ne l'est pas